share/odb_col_chl_query_filter_null.yaml

```diff
@@ -0,0 +1,15 @@
+where:
+  varno: [288,]
+query: initial_obsvalue is not null
```

Comment:
Is the indentation correct for this line? In https://github.com/JCSDA-internal/ioda/pull/1345/files the query is indented the same as varno.